resources/views/layouts/horizontal.blade.php

```diff
@@ -6,7 +6,7 @@
     @include(backpack_view('inc.head'))
 </head>
 
-<body class="{{ backpack_theme_config('classes.body') }}">
+<body class="{{ backpack_theme_config('classes.body') }}" data-bp-layout='horizontal'>
```

Comment:
a little bit inconsistency introduced here, `class=` uses double quotes `"` and you introduced a new attribute with single quotes. totally valid, just inconsistent.

Reply:
Totally, nice catch, didn't notice. Here goes:

```suggestion
<body class="{{ backpack_theme_config('classes.body') }}" data-bp-layout="horizontal">
```